src/index.js

```diff
@@ -0,0 +1,19 @@
+import './style.css';
+import * as addRemove from './modules/add-remove-list.js';
+// import * as interactive from './modules/interactive.js';
+
+addRemove.renderList();
+
```

Comment:
I think could be a good idea to insert this code inside a "DOMContentLoaded" event listener just to be sure the DOM is loaded before the list is rendered.